### ISSUE TITLE
Include AGS / Astal layers in keyboard focus hogging

### ISSUE THREAD

**Issue by jncsotk30** [hyprwm/Hyprland#9185]
### Description

Would it be possible / make sense to include AGS / Astal in the rules for the `layers_hog_keyboard_focus` property? If so, could we add this? I have a custom made launcher with AGS and would like it to keep keyboard focus even if I move my mouse.

If not, could a new property be included for configurable regex / classes for this keyboard hogging property?

Original question: https://discord.com/channels/961691461554950145/1333186387733643366/1333186704076701818

**Comment by vaxerski** [comment]
that should be done in astal. Make your layer be an exclusive keyboard focus one.

**Comment by jncsotk30** [comment]
Got it -- sorry for the erroneous ticket. Thank you Vaxxry! For anyone else, here's what was included on the window of the AGS launcher.

`keymode={Astal.Keymode.EXCLUSIVE}`

I had this, but it was set to `ON_DEMAND`.
